json-guides/microprofile-config.json

```diff
@@ -101,28 +112,38 @@
                         "import javax.inject.Inject;",
                         "import org.eclipse.microprofile.config.Config;",
                         "import org.eclipse.microprofile.config.inject.ConfigProperty;",
-                        "", 
+                        "",    
                         "public class Music-download {",
+                        "",
                         "    @Inject",
                         "    private Config config;",
                         "",
                         "    @Inject @ConfigProperty(name=\"download-url\")",
-                        "    private String downloadUrl;",
-                        "}"                
+                        "    private String download-url;",
```

Comment:
change name=download-url to name=download_url. Change String download-url to String downloadUrl.